lib/Standalone/ConvertXsmmToFunc.cpp

```diff
@@ -65,6 +64,9 @@ static SmallVector<Value, 4> getMemRefOperands(OpBuilder &b, Location loc,
 static LogicalResult buildInvokeCall(Location loc, std::string funcName,
                                      Operation *op, PatternRewriter &rewriter) {
   funcName = "xsmm_" + funcName + "_invoke";
+  if(op->getOperand(1).getType().cast<MemRefType>().getElementType().isBF16()){
```

Comment:
please drop trivial braces. MLIR does not uses it for single stmt. Let's follow a similar approach.